Source/Core/DolphinQt/Main.cpp

```diff
@@ -178,7 +178,7 @@ int main(int argc, char* argv[])
   {
     DolphinAnalytics::Instance()->ReportDolphinStart("qt");
 
-    MainWindow win{std::move(boot)};
+    MainWindow win{std::move(boot), static_cast<const char*>(options.get("movie"))};
```

Comment:
Is this cast safe when the option was not passed?

Reply:
Yes.